security/keycloak-webapp/src/test/java/io/quarkus/ts/security/keycloak/webapp/KeycloakStaticTenantWebappIT.java

```diff
@@ -0,0 +1,34 @@
+package io.quarkus.ts.security.keycloak.webapp;
+
+import static io.quarkus.test.bootstrap.KeycloakService.DEFAULT_REALM;
+import static io.quarkus.test.bootstrap.KeycloakService.DEFAULT_REALM_BASE_PATH;
+import static io.quarkus.test.bootstrap.KeycloakService.DEFAULT_REALM_FILE;
+
+import io.quarkus.test.bootstrap.KeycloakService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.KeycloakContainer;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class KeycloakStaticTenantWebappIT extends BaseWebappSecurityIT {
+
+    @KeycloakContainer(command = { "start-dev", "--import-realm", "--hostname-strict=false" })
+    static KeycloakService keycloak = new KeycloakService(DEFAULT_REALM_FILE, DEFAULT_REALM, DEFAULT_REALM_BASE_PATH);
+
+    @QuarkusApplication(classes = { OidcTenantInitializer.class, AdminResource.class, UserResource.class })
+    static RestService app = new RestService()
+            .withProperty("quarkus.oidc.auth-server-url", () -> keycloak.getRealmUrl())
+            .withProperty("quarkus.oidc.client-id", CLIENT_ID_DEFAULT)
+            .withProperties("empty.properties");
```

Comment:
I don't think this will start the Quarkus without the properties from `application.properties`. Base on https://github.com/quarkus-qe/quarkus-test-framework/issues/1510#issuecomment-2690448374 you would need to override it like this `@QuarkusApplication(classes = { OidcTenantInitializer.class, AdminResource.class, UserResource.class }, properties = "empty.properties)"`

Reply:
Np! functionality of this is hidden nicely :grin: